src/tests/CommandsTest.js

```diff
@@ -0,0 +1,55 @@
+const FileUtils = require('../utils/FileUtils.js')
+const MiscUtils = require('../utils/MiscUtils.js')
+
+const commands = []
+
+FileUtils.requireDirectory(
+  'src/commands',
+  ({ required: NewCommand }) => {
+    const { referenceCommand } = new NewCommand()
+    if (
+      typeof referenceCommand !== 'string' &&
+      !Array.isArray(referenceCommand)
+    ) {
+      commands.push(NewCommand)
+    }
+  },
+  console.error
+).catch(console.error)
+
+describe('Commands', () => {
+  it('should have no duplicate names or aliases', done => {
+    const aliases = commands.reduce((arr, NewCommand) => {
+      const { name, aliases } = new NewCommand()
+      return [...arr, name, ...(aliases || [])]
+    }, [])
+    const dupes = MiscUtils.findArrayDuplicates(aliases)
+    if (dupes.length) {
+      done(
+        new Error(
+          `The following names or aliases were found more than once: ${dupes.join(
+            ', '
+          )}`
+        )
+      )
+    } else {
+      done()
+    }
+  })
+
+  it('Should have no duplicate class names', done => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['it' is not defined.](https://app.codacy.com/gh/tobias-inc/TobiasBot/pullRequest?prid=5027451)